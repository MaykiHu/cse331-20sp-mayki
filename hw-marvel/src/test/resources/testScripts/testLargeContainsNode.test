#Tests if node is in large dataset, from marvel.tsv
#marvel.tsv should be a valid file under the marvel data folder

LoadGraph largeHeroes marvel.tsv
ContainsNode largeHeroes ANCIENT_ONE