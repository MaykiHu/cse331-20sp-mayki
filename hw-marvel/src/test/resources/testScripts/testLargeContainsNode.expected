#Tests if node is in large dataset, from marvel.tsv
#marvel.tsv should be a valid file under the marvel data folder

loaded graph largeHeroes
largeHeroes does contain ANCIENT ONE